tests/unit/providers/test_portkey_provider.py

```
@@ -1,9 +1,26 @@
+import pytest
 
 from any_llm.providers.portkey.portkey import PortkeyProvider
 from any_llm.types.completion import CompletionParams
 
 
+def test_portkey_uses_native_async_client(monkeypatch: pytest.MonkeyPatch) -> None:
+    """The provider initializes Portkey's native SDK instead of OpenAI's client."""
+    calls: dict[str, Any] = {}
+
+    class FakeAsyncPortkey:
+        def __init__(self, **kwargs: Any) -> None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)